recipes/lzfse/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "lzfse" %}
+{% set version = "1.0" %}
+{% set sha256hash = "941c2e39493f7669ee4547f3b3d9007ffb0d33c6e3d5990af5423b7b77b90bad" %}
```

Comment:
So I think this should have been ( https://github.com/lzfse/lzfse/commit/88e2d2788b4021d0b2eb9fe2d97352ae9190f128 ).